feat(localize): add option to show the key when a locale is missing a translation

pndewit opened the pull request:
## What I did

Whenever a language is missing a translation for a key, it currently shows an empty string. I would like the option to show the translation key instead. This is not only making it visible for developers that some translation is missing, but could potentially even be descriptive enough for users to figure out what it should say (at least more than an empty string) so they can continue their work for the time being.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/ing-bank/lion?pullRequest=1557) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/ing-bank/lion?pullRequest=1557) before we can accept your contribution.<br/><hr/>**Patrick de Wit** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/ing-bank/lion?pullRequest=1557) it.</sub>

tlouisse commented:
@daKmoR @jorenbroekema @gerjanvangeest any opinions on this feature?

Since it's backwards compatible, it could be a helpful debug feature imho. 
If my colleagues think as well, could you:
- rename `fallbackTranslations` to `showKeyAsFallback`
- add tests
- sign license/cla

jorenbroekema commented:
Yep, I like this! Makes it easier for users to report problems, easier to continue using the component/site by figuring out what it's meant to say roughly, so definitely agree it's good for local dev & also prod

pndewit commented:
@tlouisse I've:
- added 2 tests
- changed it to a minor update (as it is a feature and not a fix IMHO)
- squashed all commits into 1

I've also signed the license agreement about a dozen times, also used the `recheck` link, but it doesn't seem to pick it up :(

tlouisse commented:
> @tlouisse I've:
> 
> * added 2 tests
> * renamed the flag as requested
> * changed it to a minor update (as it is a feature and not a fix IMHO)
> * squashed all commits into 1
> * signed the CLA

Feature makes sense indeed. Looks good, thanks!
